cli/cmd/plugin/unmanaged-cluster/cmd/list.go

```diff
@@ -42,7 +42,7 @@ func list(cmd *cobra.Command, args []string) error {
 	tClient := tanzu.New(log)
 	clusters, err := tClient.List()
 	if err != nil {
-		return fmt.Errorf("unable to list clusters. Error: %s", err.Error())
+		return fmt.Errorf("unable to list clusters. Error: %s\n", err.Error())
```

Comment:
Should the line break be in the error message itself? It seems like it would be better to have that controlled by whatever is actually logging the error.

Reply:
The `list` func her _is_ the top most controlling function that gets passed to `cobra`, so the error formatting should happen at this level.

Reply:
Ah, missed that part!